Array_Methods_Lab.js

```diff
@@ -0,0 +1,294 @@
+/* Exercise 1:
```

Comment:
Overall, good job! My only concern is that you aren't creating functions. Lets make sure we are creating actual functions that return what we need.

Reply:
Thank you, I just fixed that